app/domain/authentication/authn_k8s/application_identity.rb

```diff
@@ -1,7 +1,7 @@
 module Authentication
   module AuthnK8s
 
-    Log = LogMessages::Authentication::AuthnK8s
+    Log = LogMessages::Authentication
```

Comment:
what?

Reply:
made some logs more general during authn-azure development so updated them respectively in this file